dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/DimensionalAggregation.java

```diff
@@ -0,0 +1,11 @@
+package org.hisp.dhis.common;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class DimensionalAggregation
```

Comment:
I am wondering if *DimensionalKeywords* could be a more descriptive name - things like "level" is not really aggregating, rather selecting, items.

Reply:
Renamed to DimensionalKeywords